errbot/backends/text.py

```diff
@@ -47,23 +48,31 @@ def __init__(self, config):
         super().__init__(config)
         log.debug("Text Backend Init.")
         self.bot_identifier = self.build_identifier('Err')
+        self.demo_mode = self.bot_config.TEXT_DEMO_MODE if hasattr(self.bot_config, 'TEXT_DEMO_MODE') else False
```

Comment:
I think we should add the `TEXT_DEMO_MODE` to the `config-template.py` somewhere (way at the bottom under misc settings?) otherwise nobody will ever find it. :smile: